src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -55,76 +55,76 @@
+    /** The unique number of target bases in the experiment, where the target sequence is usually exons etc. */
     public long TARGET_TERRITORY;
 
-    /** Target terrirtoy / bait territory.  1 == perfectly efficient, 0.5 = half of baited bases are not target. */
+    /** The ratio of TARGET_TERRITORY/BAIT_TERRITORY.  A value of 1 indicates a perfect design efficiency, while a valud of 0.5 indicates that half of bases within the bait region are not within the target region. */
     public double BAIT_DESIGN_EFFICIENCY;
 
-    /** The total number of reads in the SAM or BAM file examine. */
+    /** The total number of reads in the SAM or BAM file examined. */
     public long TOTAL_READS;
 
-    /** The number of reads that pass the vendor's filter. */
+    /** The total number of reads that pass the vendor's filter. */
     public long PF_READS;
 
     /** The number of PF reads that are not marked as duplicates. */
     public long PF_UNIQUE_READS;
 
-    /** PF reads / total reads.  The percent of reads passing filter. */
+    /** The fraction of reads passing the vendor's filter, PF_READS/TOTAL_READS.   */
     public double PCT_PF_READS;
 
-    /** PF Unique Reads / Total Reads. */
+    /** The fraction of PF_UNIQUE_READS from the TOTAL_READS, PF_UNIQUE_READS/TOTAL_READS. */
     public double PCT_PF_UQ_READS;
 
-    /** The number of PF unique reads that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF_UNIQUE_READS that aligned to the reference genome with a mapping score > 0. */
     public long PF_UQ_READS_ALIGNED;
 
-    /** PF Reads Aligned / PF Reads. */
+    /** The fraction of PF_UQ_READS_ALIGNED from the total number of PF reads. */
     public double PCT_PF_UQ_READS_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF unique bases that are aligned to the reference genome with mapping scores > 0. */
     public long PF_BASES_ALIGNED;
 
-    /** The number of bases in the PF aligned reads that are mapped to a reference base. Accounts for clipping and gaps. */
+    /** The number of bases in the PF_UQ_READS_ALIGNED reads. Accounts for clipping and gaps. */
     public long PF_UQ_BASES_ALIGNED;
 
-    /** The number of PF aligned bases that mapped to a baited region of the genome. */
+    /** The number of PF_BASES_ALIGNED that are mapped to the baited regions of the genome. */
     public long ON_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to within a fixed interval of a baited region, but not on a baited region. */
+    /** The number of PF_BASES_ALIGNED that are mapped to within a fixed interval containing a baited region, but not within the baited section per se. */
     public long NEAR_BAIT_BASES;
 
-    /** The number of PF aligned bases that mapped to neither on or near a bait. */
+    /** The number of PF_BASES_ALIGNED that are mapped distal to a baited region. */
```

Comment:
Not sure I understand the use of "distal to" here? I understand it as meaning "the farthest away from" (from bio/anatomy classes). Should it just say "away from any baited region"?